package.json

```diff
@@ -107,7 +107,7 @@
     "typescript": "^4.9.4"
   },
   "dependencies": {
-    "@tableland/evm": "^4.2.0",
+    "@tableland/evm": "^4.2.1",
```

Comment:
This PR includes the latest bug fix in main, which is not is staging yet, hence this diff.